From 872744ee8d8d56651943200d5bb3e30eac13b053 Mon Sep 17 00:00:00 2001
From: divyams <divyams@netlogicmicro.com>
Date: Wed, 13 Jul 2011 12:41:05 +0530
Subject: [PATCH 239/565] Fix for linux build warning

Fix for linux build warning, when oprofile is enabled.

Based on Broadcom SDK 2.3.

Signed-off-by: divyams <divyams@netlogicmicro.com>
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/mips/oprofile/op_model_mips_xlp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/oprofile/op_model_mips_xlp.c b/arch/mips/oprofile/op_model_mips_xlp.c
index 88dbe96..56facc9 100644
--- a/arch/mips/oprofile/op_model_mips_xlp.c
+++ b/arch/mips/oprofile/op_model_mips_xlp.c
@@ -221,7 +221,7 @@ static void nlm_common_reset_perf_counters(void)
 
 }
 
-static int __init nlm_common_init(void)
+static int nlm_common_init(void)
 {
 	int i;
 
-- 
1.8.4.93.g57e4c17

